server/routes/inPrisonEducationRecordsResolver.test.ts

```diff
@@ -140,6 +141,74 @@ describe('inPrisonEducationRecordsResolver', () => {
       // When
       const actual = mostRecentCompletedInPrisonEducationRecords(inPrisonEducationRecords, maxRecordsToReturn)
 
+      // Then
+      expect(actual).toEqual(expected)
+    })
+  })
+  describe('completedInPrisonEducationRecordsWithinLast12Months', () => {
+    it('should return completed In Prison Education records within last 12 months given InPrisonEducationRecords', () => {
+      // Given
+      const englishPrisonEducation = { ...aValidEnglishInPrisonEducation() }
+      englishPrisonEducation.courseCompleted = true
+      englishPrisonEducation.courseCompletionDate = moment('2023-07-10').toDate()
+      const mathsPrisonEducation = { ...aValidMathsInPrisonEducation() }
+      mathsPrisonEducation.courseCompleted = true
+      mathsPrisonEducation.courseCompletionDate = moment('2023-07-11').toDate()
```

Comment:
This test is going to start to fail after 10th July 2024 !!!
We've been bitten by dates in tests in the past; I'm not sure of the best way of fixing this here and now, so for the time being can you set this test to `it.skip` and leave a TODO comment saying why it's skipped please?

Reply:
How about this, which will set it to 4 months in the past dynamically 🤔 

`mathsPrisonEducation.courseCompletionDate = moment().subtract(4, 'months').toDate()`

Reply:
Yeah, OK, do that 👍 
(though we want to stop using `moment` so I feel a little uncomfortable making more use of it, but I dont know of a better way at the moment, and I dont want to introduce `date-fns` in this PR)